addons/forum-post-save/addon.json

```
@@ -0,0 +1,20 @@
+{
+  "name": "Save draft posts",
+  "description": "Saves forum posts that you haven't submitted.",
```

It may not be clear how this addon works. (_"How do you access drafts?"_) Here's a description suggestion:
```suggestion
  "description": "Saves forum posts that you haven't submitted and restores them when you come back to where you wrote them.",
```
There might also be a better way of putting this. Do you like this, @WorldLanguages?

---

Samq's looks okay

---

It's anyone's guess how long "temporary" is and what it really means here, though.

But most people will probably revisit their drafts no later than the same session or the next day so it's probably okay to say. Having an uncertain/imprecise expiration date will probably discourage users from relying on letting their unposted drafts sit there for days, which is a good thing because eventual local storage data loss is inevitable.

So, sure.